rrset.go

```diff
@@ -210,12 +210,15 @@ type SBRDataInfo struct {
 }
 
 // BackupRecord wraps the backupRecord objects of an SBPoolProfile response
+// AvailableToServe required by UltraDNS restapi
```

Comment:
Should we add comments like this for all fields? Seems odd to just doc the one.

Reply:
I think i will remove those since the changelog file already has it